content/en/docs/user-guides/how-to/plugin-management.md

```diff
@@ -38,50 +40,50 @@ To find out more about the AWS Signer plugin, please refer to their official [do
 ### Install from file system
 
 ```console
-wget https://d2hvyiie56hcat.cloudfront.net/linux/amd64/plugin/latest/notation-aws-signer-plugin.zip
+$ wget https://d2hvyiie56hcat.cloudfront.net/linux/amd64/plugin/latest/notation-aws-signer-plugin.zip
```

Comment:
It seems some command lines have `$`, some not. I would suggest to remove `$` so that users can easily copy it.

Reply:
@yizha1  If there is a command as input along with its output in a code block, it is a common convention to use `$` before the comamnd to indicate that this line is the input, which differentiates the input and output. 

Otherwise, we should split the input and output into two code blocks for all commands.

See Istio docs as an example https://istio.io/latest/docs/ambient/install/helm/

Reply:
Okay, I understand your point. I noticed that some commands in the updated document still don't have the `$` symbol because they are not meant to be examples. 

By the way, I tried the Istio documentation, and the `$` symbol will not be copied, it is really good experience.

Reply:
@yizha1 I split the input and output into two code blocks for all commands in this doc. It will be much more friendly for copy/paste. 
The code blocks on the Istio website is better. But we will need some front-end development to acheive this.